prov/sm2/src/sm2.h

```diff
@@ -108,7 +109,9 @@ enum {
  * 		   NOTE: Only grabbing the bottom 32 bits
  * 	proto - sm2 operation
  * 	sender_gid - id of msg sender
- * 	user_data - the message
+ * 	union:
```

Comment:
There's no union. Instead, the CMA protocol just casts `user_data` into `struct sm2_cma_data`.